packages/api/src/post-build.js

```diff
@@ -192,6 +236,32 @@ class PostBuildHandler {
         handleError(res, err);
       }
     });
+
+    // endpoint expects the the required buildinfo to be in req.body.metadata to already exist and be properly formatted.
+    // required keys in the req.body.metadata are the inputs for addBuild in src/add-build.js
+    this.app.post('/api/build-v1', async (req, res, next) => {
```

Comment:
since this data is fairly GitHub specific, what if we used this opportunity to move to:

```
/api/build/gh/v1
```

Indicate that this is version one of our GitHub format upload?